### PR TITLE
BI-13964: Update published_at timestamp

### PR DESCRIPTION
**Pull request by kinpang-CH**
* Add new DateUtils class
* Refactor supplied in Config
* Update mapper and tests
* Locally tested, all passes.

Resolves [BI-13964](https://companieshouse.atlassian.net/browse/BI-13964)

[BI-13964]: https://companieshouse.atlassian.net/browse/BI-13964?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by cgregoryCH — DISMISSED** [review]

File: src/main/java/uk/gov/companieshouse/exemptions/controller/ExemptionsController.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ public ResponseEntity<Void> companyExemptionsUpsert(
     public ResponseEntity<CompanyExemptions> companyExemptionsGet(
             @RequestHeader("x-request-id") String contextId,
             @PathVariable("company_number") String companyNumber) {
-        logger.info(String.format("Getting company exemptions for company number %s", companyNumber));
+        logger.info(String.format("KCP - Getting company exemptions for company number %s", companyNumber));
```
Comment (crowlandsCH) [inline src/main/java/uk/gov/companieshouse/exemptions/controller/ExemptionsController.java:55]: Leftover from local testing here looks like

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/main/java/uk/gov/companieshouse/exemptions/controller/ExemptionsController.java (hunk 1 above)
Comment (kinpang-CH) [reply src/main/java/uk/gov/companieshouse/exemptions/controller/ExemptionsController.java:55]: Nice spot, added this in to help with debugging
Removed in the latest commit.

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Review by cgregoryCH — APPROVED** [review]

**Review by crowlandsCH — APPROVED** [review]

**Pull request merged**